db/schema_7.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE birthdays
+(
+    userId TEXT NOT NULL UNIQUE,
+    date   INT  NOT NULL,
```

Comment:
rename to day (or basically anything else than `date`, since that is technically a keyword).

Reply:
in pure sql that may be the case, but not in sqlite.
But yea.. its a bad habbit :D